CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
+* reduce query creation time with shared runtimes ([#8888](https://github.com/confluentinc/ksql/pull/8888)) ([f3c5850](https://github.com/confluentinc/ksql/commit/f3c58508a379543b0e545a9cb03af1e9166760c3))
+* register schema within sandbox ([#8614](https://github.com/confluentinc/ksql/pull/8614)) ([ba572e0](https://github.com/confluentinc/ksql/commit/ba572e0e5bd8873e538d40a58153ba345eb1573d)), closes [#1394](https://github.com/confluentinc/ksql/issues/1394)
+* reinstate the old KsqlRestClient.create overload ([#8761](https://github.com/confluentinc/ksql/pull/8761)) ([ee4a1bc](https://github.com/confluentinc/ksql/commit/ee4a1bcf4ae4487e01bd4a6341feac402ffe6ddc))
+* reorder imports ([#8716](https://github.com/confluentinc/ksql/pull/8716)) ([30fd66b](https://github.com/confluentinc/ksql/commit/30fd66b3ecd26a9e37f2758b53a46d7fc514df4e))
+* resolve schema registry issue for pull query  ([#8876](https://github.com/confluentinc/ksql/pull/8876)) ([6a1c2ae](https://github.com/confluentinc/ksql/commit/6a1c2ae36d1257b050798b2a1f3cdbec4b68470d))
+* restore process fails due to DROP constraints ([#8803](https://github.com/confluentinc/ksql/pull/8803)) ([db070a2](https://github.com/confluentinc/ksql/commit/db070a211b51cfb9ef7138c9b05238ba41798f4b))
+* Set the sslFactory properly for the SR REST client. ([#8830](https://github.com/confluentinc/ksql/pull/8830)) ([e69a545](https://github.com/confluentinc/ksql/commit/e69a545a7765e9056da223075d4336d482b36614))
+* skip docker build for packaging ([#8717](https://github.com/confluentinc/ksql/pull/8717)) ([59f74e8](https://github.com/confluentinc/ksql/commit/59f74e808a63afb74f5d3c4185e3bf26f97537b6))
+* spotbugs warnings with Java 11 ([#8777](https://github.com/confluentinc/ksql/pull/8777)) ([0de47ed](https://github.com/confluentinc/ksql/commit/0de47ed7bca058f439effe9c8668933c7f43086a))
+* update rate limiting test so it's not flaky ([#8872](https://github.com/confluentinc/ksql/pull/8872)) ([466f8fe](https://github.com/confluentinc/ksql/commit/466f8fe92b5ec18209e37c511a91b633d07ca96e))
+* update restore command topic tool to work with command topic configs ([#8802](https://github.com/confluentinc/ksql/pull/8802)) ([371b200](https://github.com/confluentinc/ksql/commit/371b200d92b9fd6083bb453af74a31d4b754ecf2))
+* use blocking call and throw exception on failure ([#8819](https://github.com/confluentinc/ksql/pull/8819)) ([0e4910a](https://github.com/confluentinc/ksql/commit/0e4910a30187f63ec49de0dba3a8733d38ec4aba))
+* when a isolated query is restarted then it should stay isolated ([#8713](https://github.com/confluentinc/ksql/pull/8713)) ([33c81f8](https://github.com/confluentinc/ksql/commit/33c81f8e9eac63a8a457f1e71e9b03b32866479b))
+
+
+
+### Reverts
+
+* consistency APIs ([#8947](https://github.com/confluentinc/ksql/pull/8947)) ([a29cec7](https://github.com/confluentinc/ksql/commit/a29cec7f76cfb31cd6540ff837c13412794831eb))

```

Comment:
Was this released in a previous version? If not I think we can leave it out of the changelog.

Reply:
This should be removed from the changelog